src/components/app.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+import "../localisation"
```

Comment:
We can agree separate by empty line imports from node_modules and from local modules

Reply:
+